templates/taxi/car_detail.html

```diff
@@ -1,12 +1,31 @@
 {% extends "base.html" %}
-
 {% block content %}
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
   <h1>Drivers</h1>
   <hr>
-  <ul>
-  {% for driver in car.drivers.all %}
-    <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
-  {% endfor %}
-  </ul>
+  <table class="table table-striped">
+    <thead>
+    <tr>
+      <th>Car</th>
+      <th>Username</th>
+      <th>First Name</th>
+      <th>Last Name</th>
+      <th>Actions</th>
+    </tr>
+    </thead>
+    <tbody>
+    {% for driver in car.drivers.all %}
+      <tr>
+        <td>{{ car.model }}</td>
+        <td>{{ driver.username }}</td>
+        <td>{{ driver.first_name }}</td>
+        <td>{{ driver.last_name }}</td>
+        <td>
+          <a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>
+          <a href="{% url 'taxi:car-delete' car.id %}" class="btn btn-danger btn-sm">Delete</a>
```

Comment:
The {% url %} template tag generates URLs dynamically using the URL name defined in your urls.py file. The parameters you pass must match the named parameters expected by the URL pattern. 

Example:

`<a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>`

> Since your URL pattern uses <int:pk>, you should use pk= in the template tag:

`<a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-primary btn-sm">Update</a>`

Ensure that the href in your template matches the URL pattern, or update the URL pattern to match the parameters used in your template.